File: pset1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    int owned;  
    do
    {
        // Request User's Input
        float dollar = get_float("Change Owned: ");
        // Convert into Coin
        owned = round(dollar * 100);
    }
    while (owned <= 0);
    
    int quarter = owned / 25;
    int dimes = (owned % 25) / 10;
    int nickel = ((owned % 25) % 10) / 5;
    int pennie = ((owned % 25) % 10) %  5;
    
    printf("%d", quarter + dimes + nickel + pennie);
}

File: pset6/cash/cash.py
from cs50 import get_float

coins = [25, 10, 5, 1]


def main():

    # Get user's money from standard input
    owned = 0.0
    while (owned <= 0):
        owned = get_float("Change Owned: ")

    # Convert money into cents
    total = owned * 100
    count = 0

    # Trying a new method by using array and while loop
    for i in coins:
        # Minus and count until
        while i <= total:
            total -= i
            count += 1

    # Dispaly total coin as standard output
    print(count)


if __name__ == "__main__":
    main()